docs/book/src/faq.md

```diff
@@ -92,9 +92,38 @@ securityContext:
 ```
 However, note that this problem is fixed and will not occur if you deploy the project in high versions (maybe >= 1.22).
 
+## The error `Too long: must have at most 262144 bytes` is faced when I run `make install` to apply the CRD manifests . How to solve it? Why this error is faced?
+
+The Kubernetes API has a size limit. Then, when the apply is used the API will add the annotation last-applied-configuration with all previous configuration which is exceeding this amount. [More info][k8s-obj-creation]
+
+Therefore, you have a few options to workround this scenario such as:
+
+- You can use the option `maxDescLen=0` to remove the description of your CRDs. Following an example:
+
+```shell
+
+ .PHONY: manifests 
+ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects. 
+     # Note that the option maxDescLen=0 was added in the default scaffold in order to sort out the issue 
+     # Too long: must have at most 262144 bytes. By using kubectl apply to create / update resources an annotation 
+     # is created by K8s API to store the latest version of the resource ( kubectl.kubernetes.io/last-applied-configuration). 
+     # However, it has a size limit and if the CRD is too big with so many long descriptions as this one it will cause the failure. 
+ 	$(CONTROLLER_GEN) rbac:roleName=manager-role crd:maxDescLen=0 webhook paths="./..." output:crd:artifacts:config=config/crd/bases 
+
+You can review the design of your APIs and see if it has not more specs than should be by hurting single responsibility principle for example. So that you might to re-design them.
```

Comment:
```suggestion
**By reviewing and refactoring the API/CRD Design

Examine your APIs to ensure they're designed with a clear focus and aren't overly complex. Overcomplicated designs can violate principles like the [Single Responsibility][single-responsability] Principle, making the CRD overly bulky.
```

Reply:
The link here for: [single-responsability] is: https://en.wikipedia.org/wiki/Single-responsibility_principle

Reply:
@Sajiyah-Salat ^ missing this part